site/content/docs/private-locations/checkly-agent-guide.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Installing Checkly Agent
+weight: 52
+menu:
+  docs:
+    parent: "Private Locations"
+aliases:
+- "/docs/private-locations/checkly-agent-guide/"
+---
+
+Note: The Checkly Agent currently only supports x86/AMD64 architecture and not ARM64. This means Apple Mac M1 systems are not currently supported.
```

Comment:
```suggestion
{{<info>}}
Note: The Checkly Agent currently only supports x86/AMD64 architecture and not ARM64. This means Apple Mac M1 systems are not currently supported.
{{<info>}}
```

Reply:
It might make sense to do the same change on the "Getting started with private locations" page as well.